tests/MongoDB.Driver.Tests/UnifiedTestOperations/UnifiedEntityMap.cs

```diff
@@ -0,0 +1,484 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MongoDB.Bson;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.Events;
+using MongoDB.Driver.GridFS;
+using MongoDB.Driver.TestHelpers;
+
+namespace MongoDB.Driver.Tests.UnifiedTestOperations
+{
+    public sealed class UnifiedEntityMap : IDisposable
+    {
```

Comment:
Maybe new names would help get my point across better.

* `From` (instead of `Create`)
* `Parse` (instead of `Initialize`)

The `From` method creates a new `UnifiedEntityMap` **From** a `BsonArray` of entity definitions.

The `Parse` method **Parses** the entities defined by the `BsonArray` of `BsonDocuments`.